src/servers/src/influxdb.rs

```diff
@@ -98,8 +100,36 @@ impl TryFrom<&InfluxdbRequest> for InsertRequests {
                 writer
                     .write_ts(INFLUXDB_TIMESTAMP_COLUMN_NAME, (timestamp, precision))
                     .context(InfluxdbLinesWriteSnafu)?;
+            } else {
+                let precision = if let Some(val) = &value.precision {
+                    *val
+                } else {
+                    DEFAULT_TIME_PRECISION
+                };
```

Comment:
Can we add a small function to extract precision? L104-L108 is duplicated with L95-L99.